js/ui/sideComponent.js

```diff
@@ -78,6 +78,15 @@ class SideComponent extends GObject.Object {
         this._propertiesChangedId =
             this.proxy.connect('g-properties-changed', this._onPropertiesChanged.bind(this));
 
+        // Clicking the background (which calls overview.showApps) hides the component,
+        // so trying to open it again will call WindowManager._mapWindow(),
+        // which will hide the overview and animate the window.
+        // Note that this is not the case when opening the window picker.
+        this._desktopShownId = Main.layoutManager.connect('background-clicked', () => {
```

Comment:
Is there a reason this https://github.com/endlessm/gnome-shell/blob/eos3.5/js/ui/sideComponent.js#L60 and this https://github.com/endlessm/gnome-shell/blob/eos3.5/js/ui/sideComponent.js#L110 snippets were not brought back?

Reply:
I did not bring them back because they are already there.

Reply:
Lol sorry!